app/src/main/java/org/mifos/mobile/repositories/UserAuthRepositoryImp.kt

```diff
@@ -0,0 +1,34 @@
+package org.mifos.mobile.repositories
+
+import io.reactivex.Observable
+import okhttp3.ResponseBody
+import org.mifos.mobile.api.DataManager
+import org.mifos.mobile.models.register.RegisterPayload
+import javax.inject.Inject
+
+class UserAuthRepositoryImp @Inject constructor(private var dataManager: DataManager?) :
```

Comment:
Can we use `val` and make it not nullable? - remove the question mark, please

Reply:
done, I have a question : 
Because we are making a dependency that's being injected by di framework as non-nullable are we sure that it'll always be non-nullabe? And could there ever be a possibility that it could be non-null?

Reply:
`var` to `val`??? 😞

Reply:
😅 done